### DISCUSSION TITLE
Win and Web Support

### DISCUSSION THREAD

**Discussion by lehmannk** [leancodepl/patrol#1821] [Ideas]
I would love to use patrol in our current flutter project. However, we deploy to Win and Web and I cannot use patrol for these targets. Is there anything in the queue for these platforms?

**Comment by bartekpacia** [comment]
Hi, we have no immediate plans for that (but of course we'd like to do it in the future)

FWIW we recently made a PoC of Patrol for macOS but it's still a long way from being stable.

**Comment by RashmiShashindra** [comment]
Hello @bartekpacia ,

Any ETA for the WEB Support feature? It would be great if we can have the Web Support as well since most Flutter Apps mainly focus on Mobile and Web. Thanks!
